tensorflow/tools/ci_build/builds/test_on_install.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
ok, but would it not be better to call it "pip.sh" (because it is build_install_and_test_pip.sh but that would be annoyingly long name ;-))